OCAExplorer/src/components/Header.tsx

```diff
@@ -0,0 +1,23 @@
+import { AppBar, Toolbar, Icon } from "@mui/material";
+
+export default function Header() {
+  const displayDesktop = () => {
+
+    return (
+      <Toolbar>
+        <img src="assets/images/BCID_H_rgb_rev.svg" alt="Go to the Government of British Columbia website" style={{
+          width: '100%',
+          height: '100%',
+          flex: 1
+        }}
+        />
+      </Toolbar>
+    )
+  };
+
+    return (
+      <header>
+        <AppBar position="static" sx={{ alignItems: 'flex-start', height: "100px", flex:1 , justifyContent: 'left' }}>{displayDesktop()}</AppBar>
```

Comment:
100px might be a bit wide. Let's reduce this to maybe 64px. We should also limit the logo height to 48px.